graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -1654,7 +1654,15 @@ def _process_edge_features(
                 .get(edge_type, {})
                 .get(feat_conf.feat_name, {})
             )
-            transformer = DistFeatureTransformer(feat_conf, self.spark, json_representation)
+            # Hard Negative Transformation use case, but should be able to be reused
+            edge_mapping_dict = {
+                "edge_type": edge_type,
+                "mapping_path": f"{self.output_prefix}/raw_id_mappings/",
+                "format_name": FORMAT_NAME,
+            }
+            transformer = DistFeatureTransformer(
+                feat_conf, self.spark, json_representation, edge_mapping_dict
+            )
```

Comment:
Instead of passing `edge_mapping_dict` to all transforms we could do something like 

```
if feature_config.feat_type == "edge_dst_hard_negative":
    # Hard Negative Transformation use case, but should be able to be reused
    edge_mapping_dict = {
        "edge_type": edge_type,
        "mapping_path": f"{self.output_prefix}/raw_id_mappings/",
        "format_name": FORMAT_NAME,
    }
    feature_config.transformation_kwargs["edge_mapping_dict"] = edge_mapping_dict
```

Reply:
That makes sense. I will make the change.

Reply:
We still need to pass something here, because inside the ``DistFeatureTransformer`` class, we have no idea about the current edge_type, mapping_path, format_name, we still need to find some ways to pass them into the feature transformer class.